client/src/components/WorkSpaceCard/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import LocationOnIcon from "@material-ui/icons/LocationOn";
```

Comment:
Avoid using double quotes. Also extra new line is unwanted.

Reply:
resolved